app/infra/ai/client.py

```diff
@@ -132,86 +132,75 @@ async def chat(
         stream: bool = False,
```

Comment:
[From AI Code Reviewer][bot] Consider adding more specific error handling within the chat function now that the broad try...except block has been removed. This will improve debugging and provide more informative error messages to users.